### ISSUE TITLE
Shell toolbar has duplicate IDs

### ISSUE THREAD

**Issue by benborgers** [carbon-design-system/ibm-security#710]
## Bug - the toolbar of the `Shell` component uses the same ID for every link in the sub-menu, which isn't valid HTML. 

**Expected behavior -**

Every item has a unique ID. I believe an ID is already passed into the component for every link here: 

https://github.com/carbon-design-system/ibm-security/blob/66ab719ee54ed957adb1c791c464894d829ea569/src/components/Toolbar/Toolbar.js#L161

So maybe that could be appended to something to create the ID? 

**Actual behavior -**

Every ID in the list is `security--nav__list__item` (the default for the `NavItem` component), which results in duplicated IDs. 

<img width="1792" alt="Screen Shot 2020-08-04 at 12 03 14 PM" src="https://user-images.githubusercontent.com/30215449/89317175-056ad480-d64b-11ea-9131-11f4049f52db.png">

### Tracing

Looks like the ID would be passed into `NavItem` around here? 

https://github.com/carbon-design-system/ibm-security/blob/66ab719ee54ed957adb1c791c464894d829ea569/src/components/Toolbar/Toolbar.js#L174-L179

### Optional information

**Version -** `1.28.0`

**Comment by SimonFinney** [comment]
Thanks @benborgers ! Are you currently unable to manually supply an ID for these items? We can also generate instance IDs as a fallback

**Comment by benborgers** [comment]
@SimonFinney Oh is there a way to manually supply IDs? Looking at the source code, there doesn't seem to be a way to provide IDs to the `NavItem`s, and [the Storybook example](https://ibm-security.carbondesignsystem.com/?path=/story/patterns-shell-security--default) also has duplicated IDs (`#security--nav__list__item`).
